ouroboros-consensus/src/Ouroboros/Consensus/MiniProtocol/ChainSync/Client.hs

```diff
@@ -245,16 +250,42 @@ bracketChainSyncClient tracer ChainDbView { getIsInvalidBlock } varCandidates
 -- 6480 = 8640@ headers. The header state history will have the same length.
 --
 -- This worst case can happen when:
--- * We are more than 6480 or respectively 8640 blocks behind, bulk syncing, and
+-- - We are more than 6480 or respectively 8640 blocks behind, bulk syncing, and
```

Comment:
I haven't gone through the whole style guide yet. Do we prefer this bullet style?

Reply:
`-- *` is a header, `-- -` is a bullet list